src/app.js

```diff
@@ -1,5 +1,7 @@
 const { express, path, exphbs } = require('./reqs');
-
+const controllers = require('./controllers');
+const bodyParser = require('body-parser');
+const dbConnection = require('./database/db_connect')
```

Comment:
Why are you requiring dbconnection in your `app`?

Reply:
becuese  i have a problem to save data i enter in the form in data base 
its necessary to require dbconnection  app

Reply:
It is not. You shouldn't require your dbConnection in your app.